tests/gui/components/svg-icon.spec.js

```diff
@@ -33,9 +34,9 @@ describe('Browser: SVGIcon', function() {
 
     it('should inline the svg contents in the element', function() {
       const icon = '../../../../../lib/gui/assets/etcher.svg';
-      let iconContents = fs.readFileSync(path.join(__dirname, '../../../lib/gui/assets/etcher.svg'), {
+      let iconContents = _.split(fs.readFileSync(path.join(__dirname, '../../../lib/gui/assets/etcher.svg'), {
         encoding: 'utf8'
-      }).split(/\r?\n/);
+      }), /\r?\n/);
```

Comment:
The missing piece seems to be the directory that you used to clone the repo, which might be anything.